lib/getinfo.c

```diff
@@ -281,6 +281,30 @@ static CURLcode getinfo_offt(struct Curl_easy *data, CURLINFO info,
     *param_offt = (data->progress.flags & PGRS_UL_SIZE_KNOWN)?
       data->progress.size_ul:-1;
     break;
+#if CURL_SIZEOF_CURL_OFF_T == 8
```

Comment:
Why limit these to 64bit only?

Reply:
Okay, since it's a relative timestamp and not absolute, sure, I'll make it 32-bit as well.